pandas/core/internals/managers.py

```diff
@@ -247,16 +245,6 @@ def __nonzero__(self) -> bool:
     # Python3 compat
     __bool__ = __nonzero__
 
-    @property
-    def shape(self) -> Shape:
-        return tuple(len(ax) for ax in self.axes)
-
-    def _normalize_axis(self, axis):
-        # switch axis to follow BlockManager logic
```

Comment:
Can you leave this one in their respective classes? 
Although the code is the same, they actually have the opposite purpose (they are used in different places in AM and BM, which probably points to the fact I should have named them differently ..), and while converting more methods to normal axis logic (xref https://github.com/pandas-dev/pandas/pull/40075), the method will also change in AM code.

Reply:
sure, will update